ocrd/ocrd/workspace.py

```diff
@@ -27,6 +28,7 @@
     pushd_popd,
     MIME_TO_EXT,
     MIME_TO_PIL,
+    MIMETYPE_PAGE
 )
 
```

Comment:
Cannot comment on non-context lines, but would like to refer to `save_image_file` at the end of the file:

Please don't forget to change its default `force=True` to `force=False`.

Reply:
Good point, thx. 0eaf5f82